src/sdk/model/vac/ba-company.ts

```diff
@@ -541,4 +543,20 @@ export class BaCompany extends Entity {
     }
     return Iland.getHttp().post(`/vac-companies/${this.uuid}/actions/request-upgrade-contract`, request);
   }
+
+  /**
+   * Reset the password for a given tenant (BaCompany)
+   * @param {BaCompanyPasswordResetRequest | BaCompanyPasswordResetRequestJson} resetPasswordRequest
+   * @returns {Promise}
+   */
+  async resetVacCompanyPassword(resetPasswordRequest: BaCompanyPasswordResetRequest |
```

Comment:
why accept both? seems to add unnecessary complexity

Reply:
hum... indeed you're right. Let me change that quick.

Reply:
Fixed!